sql/internal/specutil/convert.go

```diff
@@ -53,6 +54,14 @@ type (
 	}
 )
 
+const (
+	typeView         = "view"
+	typeTable        = "table"
+	typeColumn       = "column"
+	typeSchema       = "schema"
+	typeMaterialized = "materialized"
```

Comment:
```suggestion
	typeMaterialized = "materialized view"
```

Think this is more readable / correct.